content/docs/index.md

```diff
@@ -35,3 +35,21 @@ us a ⭐ if you like the project!
 
 ✅ Contribute to CML [on GitHub](https://github.com/iterative/cml) or help us
 improve this [documentation](https://github.com/iterative/cml.dev) 🙏.
+
+## CML principles
+
+- **[GitFlow] for data science.** Use GitLab or GitHub to manage ML experiments,
+  track who trained ML models, or modified data and when. Codify data and models
+  with [DVC](/doc/cml-with-dvc) instead of pushing to a Git repo.
+
+- **Auto reports for ML experiments.** Auto-generate reports with metrics and
+  plots in each Git Pull Request. Rigorous engineering practices help your team
+  make informed, data-driven decisions.
+
+- **No additional services.** Build your own ML platform using just GitHub or
+  GitLab and your [favorite cloud providers]: AWS, Azure, GCP, or Kubernetes. No
```

Comment:
Can be BB also?

Reply:
Yes, I think BitBucket is fully supported in every feature last time I checked.